src/gui/src/lib/external-info.ts

```diff
@@ -395,6 +473,14 @@ export async function* fetchDetails(
     trackPromise(fetchDownloadsLastYear())
   }
 
+  if (githubRepoPromise) {
+    void githubRepoPromise.then(repo => {
+      trackPromise(fetchStargazerCount(repo))
+      trackPromise(fetchOpenIssuesCount(repo))
+      trackPromise(fetchOpenPullRequestCount(repo))
+    })
+  }
```

Comment:
we should not rely on having the github repo promise resolve (as that can be rate limited), instead what we want to do is extract the orgname and reponame from the `package.json` "repository" property as we have done in the `retrieveGitHubAPIUrl` function above